pyV2DL3/vegas/fillEVENTS_not_safe.py

```diff
@@ -184,17 +214,20 @@ def __fillEVENTS_not_safe__(vegasFileIO, effective_area_files,
         evt_dict["DEC"] = arr_dict["decArr"]
         evt_dict["ALT"] = arr_dict["altArr"]
         evt_dict["AZ"] = arr_dict["azArr"]
-        evt_dict["ENERGY"] = arr_dict["energyArr"]
```

Comment:
this gets moved to an if-block at line 249 - is there an else condition, or is this only available for event_class_mode.

Reply:
This is picked up elsewhere basically outside of the loop that only matters for event class mode. It's also why I put the e bias event class error